modules/ROOT/pages/dynamic-evaluate-component-reference.adoc

```diff
@@ -4,29 +4,49 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: mule, ee, esb, studio, dynamic, evaluate, expression
 
-*Available since 4.1*
-
-This Core component evaluates an expression that should result in another script, then evaluates that script for a final result. Such behavior allows you to dynamically select the script, instead of hardcoding it through the xref:transform-component-about.adoc[Transform Message Component].
+The Dynamic Evaluate component evaluates an expression to select a DataWeave script, and then executes the new script to generate a final result. This behavior enables you to dynamically select the script, instead of hardcoding it into the xref:transform-component-about.adoc[Transform Message Component].
 
 The script can use any of the usual xref:dataweave-variables-context.adoc[context variables], such as `message`, `payload`,`vars`, or `attributes`, but you can also add custom ones by providing a set of key-value pairs.
 
-The following example selects a script from a database through a `userId` query parameter and stores that script in a `userScript` variable. The `dynamic-evaluate` component then accesses the variable to invoke the script so  it can add a `name` variable from a `userName` query parameter.
+== Dynamic Evaluate Configuration
+
+[%header,cols="1,1,2,2"]
+|===
+| Field | Value | Description | Example
+
+| Expression | DataWeave expression | Specifies the expression to select a DataWeave script, and then Mule executes this script to produce a result. |
+`expression="#[vars.generateOrderScript]"`
+| Parameters | DataWeave expression | Specifies key-value pairs that should be bound to evaluate the final script. |
+`#[{joiner: ' and ', id: payload.user.id}]`
+
+|===
+
+== Example
```

Comment:
Do we really need this heading? It's a peer to a topic that actually contains it, but I think it would be fine to follow the table with the text below.

Reply:
If we keep headings like this, I think they should be more specific: Example of....